msal/tests/dev apps/NetCoreTestApp/NetCoreTestApp.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
Made it netcoreapp2.0 because it's difficult to find the netcore sdk 1.x nowadays. Also, most customers use netcore 2.x